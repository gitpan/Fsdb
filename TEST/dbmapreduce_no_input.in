#h experiment duration
# test trailing comment
