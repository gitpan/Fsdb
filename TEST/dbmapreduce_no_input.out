#fsdb experiment mean stddev pct_rsd conf_range conf_low conf_high conf_pct sum sum_squared min max n
-	-	-	-	-	-	-	0.95	0	0	-	-	0
#   | dbmapreduce -k experiment -C dbcolstats(qw(--nolog duration))
